recipes/conda-content-trust/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: conda-content-trust
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda/conda-content-trust/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 50a2732dcf3612bdff2b36171e3ebc72a74bbc258543a5aa2618218406b30a0d
+
+build:
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+  number: 0
+  noarch: python
+  entry_points:
+    - conda-content-trust=conda_content_trust.cli:cli
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - six
+    - cryptography
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - pytest-cov
+    - ruamel_yaml
+  commands:
+    - pytest -v tests
+    - conda-content-trust --help
+
+about:
+  home: https://github.com/conda/conda-content-trust
+  summary: Signing and verification tools for Conda
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - wolfv
+    - adriendelsalle
```

Comment:
Is this where one would add chenghlee & awwad?

Reply:
Yep, I added you both